clientv3/watch.go

```diff
@@ -52,6 +59,22 @@ type WatchResponse struct {
 	Canceled bool
 }
 
+// IsCreateEvent returns true if the event tells that the key is newly created.
+func IsCreateEvent(e *Event) bool {
```

Comment:
I already test this on events returned from watch chan.
Feeding in events doesn't make sense from user's point.